lib/saint.js

```diff
@@ -79,15 +79,23 @@ p.getJobSegment = function(fileId, pageNum, callback){
 	});
 }
 
-p.importJob = function(jobDescription, jobData, callback){
-	var self = this,
-			populateData =[];
-	while(jobData.length){
-		populateData.push(jobData.splice(0,10000));
+p.importJob = function(jobDescription, jobData, callback, jobId){
+	var self = this;
+	var populateData = spliceJobData(jobData);
+	function spliceJobData(jobData) {
```

Comment:
Not sure this is splicing the data, so much as reversing the order.

Reply:
spliceJobData is technically what it is doing, but it would be better if we named it with a purpose, this doesn't say why splicing is necessary. This is why reverseJobDataOrder or something like that seems to make more sense. Maybe not.

Reply:
I'm not entirely sure what the original programmer was thinking when they wrote this. I broke this out into it's own function to aid in separating the job creation from the job commit. I'll look into it more.